### PR TITLE
Bug#4284

### PR DESCRIPTION
**Pull request by ikzntsv**
Fixed small inconsistencies in meta tags

### CODE REVIEW

File: resources/views/newDesign/socialModule/_index.blade.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @include('newDesign.socialModule.facebook.set',[
-    'app_name'      => env('APP_NAME'),
```
Comment (Quicks) [inline resources/views/newDesign/socialModule/_index.blade.php]: Why you change that line?

Comment (ikzntsv) [reply resources/views/newDesign/socialModule/_index.blade.php]: screen
There are no APP_NAME in env file

Comment (Quicks) [reply resources/views/newDesign/socialModule/_index.blade.php]: I can add

**Comment by ikzntsv** [comment]
Check in now

**Pull request merged**